app/templates/frameworks/_framework_actions.html

```diff
@@ -35,8 +35,7 @@
     </a>
   {% endif %}
   <p class="browse-list-item-body">
-    Agree to the terms of the bid, provide supplier information and
-    confirm&nbsp;eligibility.
+    Confirm eligibility, agree to the application terms and provide supplier information.
```

Comment:
Confirmed - the new copy also has one word on the second line:
![screen shot 2017-02-27 at 14 48 07](https://cloud.githubusercontent.com/assets/6525554/23365688/d69d8110-fcfb-11e6-9f2c-1ff00bdd12b5.png)